src/components/Scrollable/Scrollable.tsx

```diff
@@ -160,7 +160,7 @@ export class Scrollable extends Component<ScrollableProps, State> {
     const shouldTopShadow = Boolean(shadow && scrollTop > 0);
 
     const canScroll = scrollHeight > clientHeight;
-    const hasScrolledToBottom = scrollHeight - scrollTop === clientHeight;
+    const hasScrolledToBottom = scrollHeight - scrollTop <= clientHeight + 2;
```

Comment:
Can we make 2 a constant that somehow conveys what it's suppose to do? i.e.

```suggestion
    const hasScrolledToBottom = scrollHeight - scrollTop <= clientHeight + LOW_RES_BUFFER;
```